.github/workflows/docker.yml

```diff
@@ -76,19 +76,25 @@ jobs:
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v1
 
-      - name: Login to DockerHub
+      - name: Login to GHCR
         uses: docker/login-action@v2
         with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
+          registry: ghcr.io
+          username: ${{ env.REPOSITORY }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set up Image ID
+        run: |
+          IMAGE_ID=ghcr.io/${{ env.REPOSITORY }}/xline
+          echo IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]') >> $GITHUB_ENV
```

Comment:
Why set up image ID at here, Could we set up it directly in `env`

Reply:
If we avoid hard-coding the `REPOSITORY`, we must convert the image ID to lowercase before pushing it to GHCR.